src/loading/loadingReducer.js

```diff
@@ -1,32 +0,0 @@
-/* @flow strict-local */
-import type { LoadingState, Action } from '../types';
-import {
-  DEAD_QUEUE,
-  LOGOUT,
-  ACCOUNT_SWITCH,
-  INITIAL_FETCH_START,
-  INITIAL_FETCH_COMPLETE,
-} from '../actionConstants';
-
-const initialState: LoadingState = {
-  unread: false,
-  users: false,
-};
-
-export default (state: LoadingState = initialState, action: Action): LoadingState => {
-  switch (action.type) {
-    case DEAD_QUEUE:
-    case LOGOUT:
-    case ACCOUNT_SWITCH:
-      return initialState;
```

Comment:
Note these cases -- be sure to preserve the behavior in these cases too.

Reply:
Yups, updated 👍